projects/Amlogic/devices/S905/packages/wetekdvb/package.mk

```diff
@@ -0,0 +1,41 @@
+################################################################################
```

Comment:
we have already that package https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/linux-drivers/wetekdvb/package.mk

Reply:
This is a little bit different package with wetekdvb.ko by @codesnake.

Reply:
yea but instead forging an second package wouldn't a 
``` 
if [ $LINUX = "amlogic-3.14" -o $LINUX = "amlogic-3.10" ]; then
  ...
fi
```
or similar (maybe even Device/Project ...) at `packages/linux-drivers/wetekdvb/package.mk` do the same ?